lua/obsidian/init.lua

```diff
@@ -266,7 +266,13 @@ end
 client.today = function(self)
   ---@type string
   ---@diagnostic disable-next-line: assign-type-mismatch
-  local id = os.date "%Y-%m-%d"
+  local formatted_date
+  if self.opts.daily_notes.format_date ~= nil then
+    formatted_date = self.opts.daily_notes.format_date()
```

Comment:
We'll need to use this below in `client.yesterday` as well. So the `format_date()` function should probably take an optional `time` argument just like `os.date()`.